.github/workflows/AT2.yml

```diff
@@ -66,7 +66,10 @@ jobs:
           sed -i "/\[rhel8\]/a `cat /etc/hostname`" /home/runner/_work/Trilinos/Trilinos/packages/framework/ini-files/supported-systems.ini
           printf "\n\n\n"
 
-          python ${GITHUB_WORKSPACE}/packages/framework/pr_tools/PullRequestLinuxDriverTest.py \
+          # should print unknown if $AT2_IMAGE env. variable cannot be found
```

Comment:
I feel that this is somewhat documenting the shell, we can probably lose this comment.

Reply:
My only concern here is that it _will_ be printing `unknown` until a newer image is pushed to AT2 and some other devs may not realize what's going on.  I'm OK with that so long as we get a new image built and deployed to AT2 sooner rather than later.